src/gl_fns.rs

```diff
@@ -109,43 +109,43 @@ impl Gl for GlFns {
     }
 
     fn gen_buffers(&self, n: GLsizei) -> Vec<GLuint> {
+        let mut result: Vec<_> = repeat(0 as GLuint).take(n as usize).collect();
```

Comment:
better just use `vec![0; n as usize]`

Reply:
Sure, added that.

Reply:
By the way, what do you think about using `vec![0 as _; n as _]`, it will infer all types automatically and it's even less typing.

Reply:
heheh, I don't think the first part is needed. `0` should be coerced without a cast. As for the second, I think it's fairly straightforward, so `_` is not confusing.